kratos/includes/model_part.h

```diff
@@ -1926,7 +1926,7 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     ModelPart(std::string const& NewName, IndexType NewBufferSize,VariablesList::Pointer pVariableList, Model& rOwnerModel);
 
     /// Copy constructor.
-    ModelPart(ModelPart const& rOther) = delete;
+    ModelPart(ModelPart const& rOther) = delete; // TODO: Discuss about activate it using the implementation from DeepCopyModelPart from AuxiliarModelPartUtilities
```

Comment:
I would keep it deleted. It is very inefficient if some one copying a model part just because forgot to put an &....

Reply:
Okidoki

Reply:
```suggestion
    ModelPart(ModelPart const& rOther) = delete;
```